src/test/claimtriebranching_tests.cpp

```diff
@@ -1723,7 +1723,7 @@ BOOST_AUTO_TEST_CASE(normalization_does_not_kill_sort_order)
 BOOST_AUTO_TEST_CASE(normalization_does_not_kill_expirations)
 {
     ClaimTrieChainFixture fixture;
-    fixture.setExpirationTime(3);
+    fixture.setExpirationForkHeight(800, 3, 800);
```

Comment:
I don't think this is right. Aren't those 800s supposed to be 140, the current height?

Reply:
In ClaimTrieChainFixture we don't change the consensus, so setExpirationTime was supposed to changed only expiration time, since we don't have such functionality it should be
```
fixture.setExpirationForkHeight(consensus.nExtendedClaimExpirationForkHeight, 3, consensus.nExtendedClaimExpirationTime)
```